app/helpers/placements/support/organisations_helper.rb

```diff
@@ -11,6 +11,19 @@ def no_results(search = "", filters = [])
     end
   end
 
+  def organisation_url(organisation)
+    case organisation.class.name
+    when "Placements::School"
+      placements_support_school_path(organisation)
+    when "School"
+      placements_support_school_path(organisation)
+    when "Placements::Provider"
+      placements_support_provider_path(organisation)
+    when "Provider"
+      placements_support_provider_path(organisation)
+    end
+  end
```

Comment:
I _think_ this should work because you can [match an object's type in case statements](https://batsov.com/articles/2012/10/14/ruby-tip-number-3-matching-on-an-objects-class-in-a-case-expression/).

```suggestion
  def organisation_url(organisation)
    case organisation
    when School
      placements_support_school_path(organisation)
    when Provider
      placements_support_provider_path(organisation)
    end
  end
```

I'm hoping it will handle both the child class (e.g. `Placements::School`) and the parent class (e.g. `School`) due to them both being a type of `School`.

Reply:
Sweet, I'll give it a go